### PR TITLE
Small various edits and cleanups

### PR DESCRIPTION
**Pull request by Coldfirex**
Remove dead links and cleanup.

### CODE REVIEW

File: content/community/links.html [hunk 1]
```diff
@@ -22,20 +20,17 @@
 <li><a href="http://www.besly.de/" target=_blank>BeSly</a>: German BeOS, Haiku and ZETA knowledge base. Tutorials, translations 
 and more in several languages.</li>
 <li><a href="http://betips.net/" target=_blank>BeTips</a>: The Haiku/BeOS Tip Server.</li>
-<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source  projects. <a href="https://haikuarchives.github.io/software.html">Software Archives <a></li>
+<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source projects.</li>
+<li><a href="https://haikuarchives.github.io/software.html" target=_blank>HaikuArchives Index</a>: A partial index of the Haiku Archives.</li>
 <li><a href="http://haiku-gazette.blogspot.com" target=_blank>Haiku Gazette</a>: Haiku news in German.</li>
 <li><a href="http://www.haiku-os.it" target=_blank>Haiku OS Italia</a>: News for Italian Haiku users.</li>
```
Comment (pulkomandy) [inline content/community/links.html:26]: This should be removed, the original website is down and what's there now is used for SEO hacks for an online casino.

File: content/community/links.html [hunk 2]
```diff
@@ -9,9 +9,7 @@
 <p><strong>Social Networking</strong></p>
 
 <ul>
- <li><a href="https://plus.google.com/communities/116050756631351429142" title="Google Plus Community Group">Google Plus Community Group</a></li>
  <li><a href="https://www.facebook.com/groups/haikuosgroup/" title="Facebook Community Group">Facebook Community Group</a></li>
- <li><a href="https://www.linkedin.com/groups?gid=165215" title="Linkedin Community Group">Linkedin Community Group</a></li>
```
Comment (pulkomandy) [inline content/community/links.html:14]: Correct link appears to be https://www.linkedin.com/groups/165215/

File: content/community/links.html [hunk 3]
```diff
@@ -22,20 +20,17 @@
 <li><a href="http://www.besly.de/" target=_blank>BeSly</a>: German BeOS, Haiku and ZETA knowledge base. Tutorials, translations 
 and more in several languages.</li>
 <li><a href="http://betips.net/" target=_blank>BeTips</a>: The Haiku/BeOS Tip Server.</li>
-<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source  projects. <a href="https://haikuarchives.github.io/software.html">Software Archives <a></li>
+<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source projects.</li>
+<li><a href="https://haikuarchives.github.io/software.html" target=_blank>HaikuArchives Index</a>: A partial index of the Haiku Archives.</li>
```
Comment (pulkomandy) [inline content/community/links.html:24]: Is it useful to link this? No one is maintaining it and there isn't any useful info there besides what's already in the github?

File: content/community/links.html (hunk 2 above)
Comment (Coldfirex) [reply content/community/links.html:14]: Done, thanks!

File: content/community/links.html (hunk 3 above)
Comment (Coldfirex) [reply content/community/links.html]: Removed.

File: content/community/links.html (hunk 1 above)
Comment (pulkomandy) [reply content/community/links.html:26]: The one I commented on, http://www.haiku-os.it

Comment (Coldfirex) [reply content/community/links.html]: Removed.  I can see it from my side, but I have a Pi-Hole setup that may be blocking it.

**Pull request merged**